contracts/cw3-flex-multisig/src/msg.rs

```diff
@@ -109,7 +109,7 @@ pub enum ExecuteMsg {
     Propose {
         title: String,
         description: String,
-        msgs: Vec<CosmosMsg<Empty>>,
+        msgs: Vec<SubMsg<Empty>>,
```

Comment:
Here we can keep `CosmosMsg` too. It can be wrapped when the message is actually executed.